std/meta.d

```diff
@@ -1,12 +1,12 @@
 // Written in the D programming language.
 
 /**
- * Templates to manipulate template argument lists (also known as alias tuples).
+ * Templates to manipulate template argument lists (also known as type lists).
```

Comment:
Next paragraph should state that this can contain types, symbols (aliases), and values.